File: MPI/mpicheck/tests/functions/3.0/MPI_Win_create_dynamic.c
#include <mpi.h>
int main(char argc, char**argv)
{
    /* vars */
    MPI_Info var_0;
    MPI_Comm var_1;
    MPI_Win *var_2;
    int ret;
    /* calls */
    ret = MPI_Win_create_dynamic(var_0, var_1, var_2);
    ret = PMPI_Win_create_dynamic(var_0, var_1, var_2);
    return 0;
}

File: MPI/mpicheck/tests/functions/3.0/MPI_Win_create_dynamic.f90

        
        program main
        use mpi
        
        INTEGER var_0
       INTEGER var_1
       INTEGER var_2
       INTEGER var_3
        call mpi_win_create_dynamic(var_0, var_1, var_2, var_3)
       call pmpi_win_create_dynamic(var_0, var_1, var_2, var_3)
        end program main
    